### PR TITLE
Luxor ambiguity clarification in howto and tutorial 1

### PR DESCRIPTION
**Pull request by Wikunia**
## PR Checklist

If you are contributing to `Javis.jl`, please make sure you are able to check off each item on this list:

- [ ] Did I update `CHANGELOG.md` with whatever changes/features I added with this PR?
- [ ] Did I make sure to only change the part of the file where I introduced a new change/feature?
- [ ] Did I cover all corner cases to be close to 100% test coverage (if applicable)?
- [ ] Did I properly add Javis dependencies to the `Project.toml` + set an upper bound of the dependency (if applicable)?
- [ ] Did I properly add test dependencies to the `test` directory (if applicable)?
- [ ] Did I check relevant tutorials that may be affected by changes in this PR?
   -  all above not necessary as it's a doc change
- [x] Did I clearly articulate why this PR was made the way it was and how it was made?

**Link to relevant issue(s)**
No issue for this just from the Zulip channel

**How did you address these issues with this PR? What methods did you use?**
I've updated the HowTo section by adding a 
```
How can I avoid ambiguity errors with Luxor.jl?
```
section.

Additionally I added two sentences to the first tutorial to make clear that `using Luxor` should be avoided inside `Javis` scripts.

### CODE REVIEW

**Comment by TheCedarPrince** [comment]
@Wikunia - I added some clarifying language and fixed some grammar. Otherwise, if it looks good to you, feel free to merge it. :smile:

**Review by TheCedarPrince — APPROVED** [review]

File: docs/src/howto.md [hunk 1]
```diff
@@ -19,14 +19,15 @@ Background(1:100, ground)
...
 ```
 
-## How can I avoid ambiguity errors with Luxor.jl?
+## Why are all my `Javis` functions undefined? 
 
-You might have noticed that [Luxor.jl](https://github.com/JuliaGraphics/Luxor.jl) provides the drawing functions that Javis is using.
-Javis is basically just one more package on top of it which provides functions to animate the static images you can create with Luxor.
-As one can't use Javis without using Luxor we decided to reexport all functions that Luxor exports. This means you don't have to and
-**shouldn't** call `using Luxor` as this will result in ambiguity errors. 
+If you have worked with the animation package [`Luxor.jl`](https://github.com/JuliaGraphics/Luxor.jl), you will be happy to that it provides all the drawing functions that `Javis` uses.
```
Comment (Wikunia) [inline docs/src/howto.md:24]: *happy to **see** ?

File: docs/src/howto.md [hunk 2]
```diff
@@ -19,14 +19,15 @@ Background(1:100, ground)
...
+`Javis` is basically an abstraction layer built on on top of `Luxor.jl` which provides functions to animate the static images you can create with `Luxor` more easily.
+As one can't use `Javis` without using Luxor we decided to reexport all functions that `Luxor` exports.
 
-Another reason we do this is the case that we sometimes need to save extra functionality when you call certain functions to have more useful information we need to create better animations. 
+This means you **should not** call `using Luxor` in scripts that use `Javis`.
+Otherwise it will result in ambiguity errors such as all the `Javis` functions being undefined when you try to run a script with `Javis` in it or other strange conflicts.
```
Comment (Wikunia) [inline docs/src/howto.md:29]: *functions **are** being undefined?

**Review by Wikunia — COMMENTED** [review]
Just two grammar questions :wink:

File: docs/src/howto.md (hunk 1 above)
Comment (cormullion) [reply docs/src/howto.md:24]: “animation package” is a bit of a stretch 😂

Comment (Wikunia) [reply docs/src/howto.md:24]: Agree. I would call Javis an animation package and Luxor a drawing package.

**Comment by TheCedarPrince** [comment]
Done! Thanks for your thoughts and input @cormullion and your initial work on this @Wikunia ! :smile:
